src/Unicode-Initialization/ManifestUnicodeInitialization.class.st

```diff
@@ -1,7 +1,10 @@
+"
+Unicode initialization 
```

Comment:
I'm not sure this kind of comment is useful. It gives the feeling that it's just sweeping the dust under the carpet. 

Maybe something like this would be better:

"I extend classes from other Pharo packages in order to give them access to mappings of Unicode values"